Makefile

```diff
@@ -3,7 +3,7 @@
 .DEFAULT_GOAL := help
 
 # For opening files in a browser. Use like: $(BROWSER)relative/path/to/file.html
-BROWSER := python -m webbrowser file://$(CURDIR)/
```

Comment:
This didn't work, since it can't concatenate the two paths. Most of our projects do it like this:

```
define BROWSER_PYSCRIPT
import os, webbrowser, sys
try:
        from urllib import pathname2url
except:
        from urllib.request import pathname2url

webbrowser.open("file://" + pathname2url(os.path.abspath(sys.argv[1])))
endef
export BROWSER_PYSCRIPT
BROWSER := python -c "$$BROWSER_PYSCRIPT"
```

And maybe that would be better, simply for consistency's sake, than what's in my PR.